functions/pipeline/images/__init__.py

```diff
@@ -14,6 +14,7 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
     user_name = req.params.get('userName')
     tag_status = req.params.get('tagStatus')
     image_ids = req.params.get('imageId')
+    checkout = req.params.get('vott')
```

Comment:
@samiyaakhtar instead of "vott" we can have "checkOut" so that the API doesn't have to know anything about VOTT

Reply:
@andrebriggs sure, and yeah i was able to test and verify it locally.